bgpd/rfapi/vnc_debug.c

```diff
@@ -95,6 +95,7 @@ DEFUN (no_debug_bgp_vnc,
        "<no debug|undebug> bgp vnc <rfapi-query|import-bi-attach|import-del-remote|verbose>",
```

Comment:
Is 'undebug' used for anything else? why are we deviating from the 'no' convention here? Is it necessary to add another node at the top level for existing functionality?